tools/cdk-build-tools/lib/package-info.ts

```diff
@@ -89,6 +89,14 @@ export function packageCompiler(compilers: CompilerOverrides): string[] {
   }
 }
 
+/**
+ * Return the command defined in scripts.gen if exists
+ */
+export function genScript(): string | undefined {
+  return currentPackageJson().scripts.gen;
```

Comment:
```suggestion
  return currentPackageJson().scripts?.gen;
```

Not likely but *possible* that `scripts` is empty.